src/Octoshift/Services/ReclaimService.cs

```diff
@@ -136,6 +136,8 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
 
         var githubOrgId = await _githubApi.GetOrganizationId(githubTargetOrg);
 
+        // org.enterprise_managed_user_enabled?
```

Comment:
I think this is just residual note from when I working on this. I think we can delete.